itest/universe_test.go

```diff
@@ -497,6 +498,52 @@ func testUniverseFederation(t *harnessTest) {
 		require.NoError(t.t, waitErr)
 	}
 
+	// Check that we can fetch the group anchor from the federation server
+	// by its asset ID in addition to its group key.
+	groupedAsset := fn.Filter(newAssets, func(asset *taprpc.Asset) bool {
+		return asset.AssetGroup != nil
+	})
+	require.Len(t.t, groupedAsset, 1)
+
+	// Query for the group anchor with only the asset ID.
+	uniIDNoGroupKey := &unirpc.ID{
+		Id: &unirpc.ID_AssetId{
+			AssetId: groupedAsset[0].AssetGenesis.AssetId,
+		},
+	}
+	var groupUniRoots *unirpc.QueryRootResponse
+
+	waitErr := wait.NoError(func() error {
+		var err error
+		groupUniRoots, err = t.tapd.QueryAssetRoots(
```

Comment:
Shouldn't this succeed on the very first try? Why do we need `wait.NoError` wrapped around the call.